### PR TITLE
Fix windows fluent-bit feature to use resource optimization flag

### PR DESCRIPTION
**Pull request by wanlonghenry**
Fix windows fluent-bit feature to use resource optimization flag

issue 1: chart values for AZMON_WINDOWS_FLUENT_BIT_DISABLED was removed from rs pod
 
issue 2: as the AZMON_WINDOWS_FLUENT_BIT_DISABLED was set for windows, this value is always trues in win

### CODE REVIEW

File: build/common/installer/scripts/fluent-bit-conf-customizer.rb [hunk 1]
```diff
@@ -132,7 +132,8 @@ def substituteFluentBitPlaceHolders
     stacktraceLanguages = ENV["AZMON_MULTILINE_LANGUAGES"]
     resourceOptimizationEnabled = ENV["AZMON_RESOURCE_OPTIMIZATION_ENABLED"]
     enableCustomMetrics = ENV["ENABLE_CUSTOM_METRICS"]
-    windowsFluentBitDisabled = ENV["AZMON_WINDOWS_FLUENT_BIT_DISABLED"]
+    # windowsFluentBitDisabled = ENV["AZMON_WINDOWS_FLUENT_BIT_DISABLED"]
```
Comment (ganga1980) [inline build/common/installer/scripts/fluent-bit-conf-customizer.rb]: lets remove the commented code.

File: kubernetes/linux/main.sh [hunk 2]
```diff
@@ -1060,9 +1060,10 @@ if [ ! -f /etc/cron.d/ci-agent ]; then
       echo "*/5 * * * * root /usr/sbin/logrotate -s /var/lib/logrotate/ci-agent-status /etc/logrotate.d/ci-agent >/dev/null 2>&1" >/etc/cron.d/ci-agent
 fi
 
-setGlobalEnvVar AZMON_WINDOWS_FLUENT_BIT_DISABLED "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}"
+# setGlobalEnvVar AZMON_WINDOWS_FLUENT_BIT_DISABLED "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}"
```
Comment (ganga1980) [inline kubernetes/linux/main.sh]: same here.

File: kubernetes/linux/main.sh [hunk 3]
```diff
@@ -1060,9 +1060,10 @@ if [ ! -f /etc/cron.d/ci-agent ]; then
...
+setGlobalEnvVar AZMON_WINDOWS_FLUENT_BIT_DISABLED "$( [ "${AZMON_RESOURCE_OPTIMIZATION_ENABLED}" == "true" ] && echo "false" || echo "true" )"
 if [ "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}" == "true" ] || [ -z "${AZMON_WINDOWS_FLUENT_BIT_DISABLED}" ] || [ "${USING_AAD_MSI_AUTH}" != "true" ] || [ "${RS_GENEVA_LOGS_INTEGRATION}" == "true" ]; then
-      if [ -e "/etc/config/kube.conf" ]; then
```
Comment (ganga1980) [inline kubernetes/linux/main.sh:1065]: This check - [ "${RS_GENEVA_LOGS_INTEGRATION}" == "true" ]; needs to be removed for cosmic to get the metrics collected through Geneva. let me test this change quickly.

Comment (ganga1980) [inline kubernetes/linux/main.sh:1065]: This change needs to be reverted.

Comment (wanlonghenry) [reply kubernetes/linux/main.sh:1065]: branch [longw/win-ama-resource-optimization-flag-cosmic-geneva]

**Pull request closed without merging**